### DISCUSSION TITLE
Darkest Dark Theme  and office integration  do not respond

### DISCUSSION THREAD

**Discussion by nicolanicoletti** [dbeaver/dbeaver#18517] [Q&A]
### Description

I had dbaver 22.2.5 with the "Darkest Dark Theme " and office integration plugin . I have upgraded to 22.3 and it doesn' t allow me to install them because  sites do not respond. I have downgraded to the previous version and have the same problem . 

### DBeaver Version

22.2.5 and 22.3

### Operating System

windows

### Steps to reproduce

1 install dbaver
2 install new software

**Comment by nicolanicoletti** [comment]
i have reinstalled 22.3 with configuration reset option and now it work correctly.
Please close this thread

**Comment by E1izabeth** [comment]
Thank you for informing us that the issue has gone for you.
